examples/test-umbrella/Readme.md

```diff
@@ -28,7 +28,7 @@ Start the topology on Mininet.
 $ cd endeavour && python examples/test-umbrella/mininet/simple_sdx.py
 ```
     
-In another terminal session.
+In another terminal session. Before run ryu-manager, correct the path in umbrella/fabric.cfg where the cloned endeavour directory is placed.
```

Comment:
It means that the path in umbrella/fabric.cfg (/home/vagrant/endeavour/umbrella/fabric.json) has to be updated with actual path of the endeavour repo directory in a local machine. But, yes, the sentence seems not clear enough.